src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -119,6 +119,7 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
               match e with
               | GetError.OneWayToSource -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Binding {BindingName} is read-only", nameChain, binder.Name)
               | GetError.SubModelSelectedItem d -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Failed to find an element of the SubModelSeq binding {SubModelSeqBindingName} with ID {ID} in the getter for the binding {BindingName}", d.NameChain, d.SubModelSeqBindingName, d.Id, binder.Name)
+              | GetError.ToNullError (ValueOption.ToNullError.ValueCannotBeNull nonNullTypeName) -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Binding {BindingName} is null, but type {Type} is non-nullable", nameChain, binder.Name, nonNullTypeName)
```

Comment:
This message can probably be improved, but it is a bit difficult to think about this while this error is currently not possible.  It is good enough though.  Let's try to remember to reconsider this message in the future.